cobalt/browser/h5vcc_system/public/mojom/h5vcc_system.mojom

```diff
@@ -14,11 +14,22 @@
 
 module h5vcc_system.mojom;
 
+enum UserOnExitStrategy {
+  kStop,
+  kConceal,
+  kNoExit,
+};
+
 // The browser process must provide an implementation of this interface so that
 // the renderer process can implement the H5vccSystem Blink API.
 interface H5vccSystem {
   // Get the advertisingId for the device.
   GetAdvertisingId() => (string advertising_id);
   // Get the limitAdTracking for the device.
   GetLimitAdTracking() => (bool limitAdTracking);
+
+  // Returns the platform-specific user on exit strategy.
+  GetUserOnExitStrategy() => (UserOnExitStrategy userOnExitStrategy);
+  // Platform-specific conceal or stop depending on strategy.
+  ConcealOrStop() => ();
```

Comment:
Now that the IDL has changed to having only a command `Promise<boolean> concealOrStop();` and two events, keeping these two calls that have to be called in a back to back fashion doesn't seem necessary; instead IIUC we can just have a `ApplyUserOnExitStrategy() => (UserOnExitStrategy userOnExitStrategy);` that does the trick _and_ returns the current strategy in one hop. WDYT?

Reply:
> The client would want to execute logic before conceal or before stop.

I see your point. Does the client `strategy` change along the browser lifetime? Or is it always the same? If it's the same, and is known at link time, could we retrieve it in blink's `H5vccSystem` ctor and keep it there?  That would avoid the round trip `concealOrStop()` -> mojo `GetUserOnExitStrategy()` -> `OnGetUserOnExitStrategy()` every time.

Reply:
Ping ^^